src/base/Assert.hh

```diff
@@ -91,7 +118,7 @@
         if (CELER_UNLIKELY(!(COND)))                                  \
         {                                                             \
             std::ostringstream celer_runtime_msg_;                    \
-            celer_runtime_msg_ << MSG;                                \
+            celer_runtime_msg_ MSG;                                   \
```

Comment:
Looks like there are some trailing spaces here.

Reply:
Hah, I thought the same thing when I looked over the diff -- but because this is a macro there's a backslash way at the end of the line.